### DISCUSSION TITLE
Can I use damask-3.0.0 version to calculate plastic fracture of metals?

### DISCUSSION THREAD

**Discussion by SunW317** [damask-multiphysics/DAMASK#26] [Q&A]
I only see the code for brittle fracture in the official documentation，Are there any examples of plastic fracture of metals that I can learn from?

**Comment by hyharry** [comment]
I am not sure what do you mean by plastic fracture. Maybe you can try out using a isobrittle (or anisobrittle) damage model together with plasticity law.

A good reference to check out is [this paper](https://www.sciencedirect.com/science/article/pii/S0045782516303267)

**Reply by SunW317** [reply]
Thank you for your reply.I didn't find the source_damage_anisoDuctile module in the damask-3.0.0 version of the source code, which calculates ductile damage, so I would like to ask if this module was removed from the damask-3.0.0 version?

**Reply by hyharry** [reply]
we only have `anisobrittle` model, your `source_damage_anisoDuctile` module seems like a customized implementation, pls contact your source provider.

**Reply by SunW317** [reply]
Thank you very much for your reply, can you tell me if the version upgrade between damask-2.0.3 and damask-3.0.0 will result in a difference in functionality? Especially the ductile damage module.

**Reply by hyharry** [reply]
v2.0.3 to v3.0.0 is a major version update, so there are a bunch of essential updates (corrections) in the functionalities. If you are particularly interested in damage module, you should find the corresponding files and compare them yourself. Above all I advice you to use the newer version which will be more stable (since it is tested on various common usage environments).

**Reply by sharanroongta** [reply]
> Thank you for your reply.I didn't find the source_damage_anisoDuctile module in the damask-3.0.0 version of the source code, which calculates ductile damage, so I would like to ask if this module was removed from the damask-3.0.0 version?

Yes, ductile damage model was removed from version 3
A new improved damage model is in the pipeline by @hyharry . We hope to integrate it soon in the upcoming versions

**Reply by CSU-Gao** [reply]
> Thank you very much for your reply, can you tell me if the version upgrade between damask-2.0.3 and damask-3.0.0 will result in a difference in functionality? Especially the ductile damage module.

I also want to call damage module during simulation by using damask-2.0.3. However, no matter what type of damage I chose in material.config file, damage_none is called during simulation. I think this results may be caused by the wrong input in  material.config file. Therefore, I hope to have a look on your material.config file which can be used to call the anisobrittle damage model. Thinks.
